setup.py

```diff
@@ -30,12 +32,23 @@ def main() -> None:
     """Builds python package used by Google Cloud Dataflow workers."""
     # Configure the required packages and scripts to install.
     with open('requirements.txt', encoding='utf-8') as requirements_txt: # pylint: disable=replace-disallowed-function-calls
+        requirements_content = requirements_txt.read()
+        modified_requirements_content = re.sub(
+            r'^\s*--hash=sha256:.*$|\\$',
+            '',
+            requirements_content,
+            flags=re.MULTILINE
+        )
```

Comment:
Can we add a small comment explaining why we need this substitution in the requirements.txt file? Let me know if you think it is not needed, we can discuss it.

Thanks

Reply:
we need to remove the generated hashes within the requirements.txt before parsing it to pkg_resources - 
pkg_resources expects requirements to be specified per PEP 508 ([doc](https://setuptools.pypa.io/en/latest/pkg_resources.html#requirements-parsing)), which does not include the –hashes command-line argument that we use for pip.

please let me know to include this info in the `setup.py`

Reply:
I think it will be good to add a small comment regarding this.
Thanks

Reply:
done @lkbhitesh07 Thanks!